core/consensus/src/lib.rs

```diff
@@ -100,7 +100,63 @@ pub enum ConsensusError {
 
     /// The synchronous block does not pass the checks.
     #[display(fmt = "Synchronization {} block error", _0)]
-    SyncBlockHashErr(u64),
+    VerifyBlockHeaderPreBlockHashErr(u64),
```

Comment:
Remove the `err` suffix from all enumerated members, because rust's error name is typical `XXError`, adding the` err` suffix to all enumerated members is redundant.

Reply:
removed